docs/level_0/first_simple_example.md

```diff
@@ -0,0 +1,209 @@
+# First simple Example
+
+In this example we describe how to deploy an nginx with the help of Landscaper.
+
+We describe how to develop a component containing a Component Descriptor and a Blueprint with a DeployItem.
```

Comment:
Since this is the first example after reading the basic concept, maybe provide a short summary of the terms when they are used, e.g.:
Blueprint (collected set of installation tasks)

Reply:
we have added such a description